src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -953,4 +953,7 @@ Do you want to proceed?</value>
   <data name="An_error_occurred_while_generating_documentation_for_this_code" xml:space="preserve">
     <value>An error occurred while generating documentation for this code.</value>
   </data>
+  <data name="Describe_with_Copilot_is_unavailable_since_the_referenced_document_is_excluded_by_your_organization" xml:space="preserve">
+    <value>Describe with Copilot is unavailable since the referenced document is excluded by your organization</value>
```

Comment:
Do we need quotes here?

Reply:
Quotes surrounding the value? I don't know why we would?

Reply:
No, I meant something like

```suggestion
    <value>'Describe with Copilot' is unavailable since the referenced document is excluded by your organization</value>
```

Because there's this a few lines above

```xml
<data name="Computing_Encapsulate_Field_information" xml:space="preserve">
    <value>Computing 'Encapsulate Field' information</value>
  </data>
```

Reply:
screenshot added to original post